fastlane/Jetpack-Fastfile

```diff
@@ -145,7 +145,7 @@ lane :update_jetpack_appstore_strings do |options|
     'screenshot-text-4' => File.join(source_metadata_folder, 'promo_screenshot_4.txt'),
     'screenshot-text-5' => File.join(source_metadata_folder, 'promo_screenshot_5.txt'),
     'screenshot-text-6' => File.join(source_metadata_folder, 'promo_screenshot_6.txt'),
-    'app-store-name' => File.join(source_metadata_folder, 'app_name.txt')
+    app_store_name: File.join(source_metadata_folder, 'name.txt')
```

Comment:
I updated this key used in the `Jetpack-Fastfile` to be the same as the WordPress one. It's safe to do this update because the value was _never used_ since it wasn't originally in the Jetpack `.po` file. This can be verified by noticing there's no `app-store-name` on the Jetpack GlotPress page. E.g.:

![image](https://user-images.githubusercontent.com/1218433/157783073-4752e883-627d-44a3-8cee-a94e1307467c.png)

The same goes for the file name. Notice the rename diff just beneath this one.

Reply:
I double-checked this in GlotPress too, because I've been trapped by that case of "look at the GlotPress page to check if the key exists" in the past… and forgetting to look at page 2 😇  (being so used to have the GP project for metadata having so fewer entries than the main project). But unlike for WP metadata project, the Jetpack metadata project on GP fits everything in one page, so no risk to fall into that trap here after all! 😅 

> The same goes for the file name. Notice the rename diff just beneath this one.

Nice catch 👍